velox/exec/OperatorUtils.h

```diff
@@ -70,11 +72,22 @@ VectorPtr wrapChild(
     const VectorPtr& child,
     BufferPtr nulls = nullptr);
 
-// Wraps all children of the specified row vector into a dictionary using
-// specified mapping. Returns vector as-is if mapping is null.
+/// Wraps all children of the specified row vector into a dictionary using
+/// specified mapping. Returns vector as-is if mapping is null.
 RowVectorPtr
 wrap(vector_size_t size, BufferPtr mapping, const RowVectorPtr& vector);
 
+/// Wraps 'childVectors' into a dictionary with 'rowType' using specified
+/// mapping. Returns an empty vector if mapping is null. This is different than
+/// 'wrap' which takes an input vector. The latter returns the input vector if
+/// mapping is null.
+RowVectorPtr wrap(
```

Comment:
It feels a bit strange to have another method with slightly different semantics but the same name. How about name "wrapWithEmptyResults"(...) for distinction ?